source/common/tls/default_tls_certificate_selector.cc

```diff
@@ -2,6 +2,8 @@
 
 #include "source/common/tls/utility.h"
 
+#include "openssl/evp.h"
```

Comment:
I don't think this is needed anymore since the evp code moved into the context constructor